src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1656,6 +1687,7 @@ static void uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
     md->super.reg_nonblock_mem_types = UCS_BIT(UCS_MEMORY_TYPE_HOST);
     md->super.gva_mem_types          = UCS_BIT(UCS_MEMORY_TYPE_HOST);
 
+    uct_ib_mlx5_devx_check_odp_ro(md);
```

Comment:
maybe no need a separate function? since it's already inside check_odp
or maybe check it on demand, the first time GVA key is registered?